drivers/disk/usdhc.c

```diff
@@ -840,11 +845,17 @@ static int usdhc_Internal_dma_cfg(struct usdhc_priv *priv,
 		base->ADMA_SYS_ADDR = (uint32_t)(dma_cfg->adma_table);
 	}
 
+#if (defined(FSL_FEATURE_USDHC_HAS_NO_RW_BURST_LEN) && FSL_FEATURE_USDHC_HAS_NO_RW_BURST_LEN)
+	/* select DMA mode and config the burst length */
+	base->PROT_CTRL &= ~(USDHC_PROT_CTRL_DMASEL_MASK);
+	base->PROT_CTRL |= USDHC_PROT_CTRL_DMASEL(dma_cfg->dma_mode);
```

Comment:
Comment suggest setting burst length but is this really setting a burst length since this is inside the "HAS_NO_RW_BURST_LEN"?

Reply:
It isn't, I missed that change. Fixed in latest push